draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -217,13 +203,42 @@ The TTL field's value is not defined by this memo.  Catalog zones are
 for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers.
 
-# Properties {#properties}
+
+## Global Properties
+
+Apart from catalog zone metadata stored at the apex (NS, SOA and the like), catalog zone information is stored in the form of "properties".
+
+This specification defines a number of so-called properties that implementations MUST support,
+as well as a mechanism to allow implementors to store additional information in the catalog zone (= extension properties, (#customproperties)).
+The meaning of such extension properties is determined by the implementation in question.
+
+Some properties are defined at the global level; others are scoped to apply only to a specific member zone.
+This section deals with global properties; member-specific properties are described in (#properties).
+
+More properties may be defined in future documents.
+
+### Schema Version (`version` property)
+
+The catalog zone schema version is specified by an integer value embedded in a TXT RR named `version.$CATZ`.
+All catalog zones MUST have a TXT RRset named `version.$CATZ` with at least one RR.
```

Comment:
There can be only one schema version in a catalog zone. Multiple version RRs will make the catalog invalid.

Reply:
```suggestion
All catalog zones MUST have a TXT RRset named `version.$CATZ` with exactly one RR.
```